mentors/delphi/0.json

```diff
@@ -1 +1,10 @@
-[]
+[
+  {
+    "github_username": "Neftali",
+    "name": "Germán Estévez -Neftalí-",
+    "link_text": Personal website about Delphi,
+    "link_url": https://neftali.clubdelphi.com/,
```

Comment:
```suggestion
    "link_url": "https://neftali.clubdelphi.com/",
```

Reply:
OK.
Sorry, I haven't worked with Github directly on the web